changelog/19738.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
You can delete the changelog and apply the `pr/no-changelog` label to skip the CI check - we only do changelogs for changes that affect the released binaries (i.e. not for docs, CI, codeowners updates etc).